test/SlowTests/Issues/RavenDB_21233.cs

```diff
@@ -0,0 +1,132 @@
+﻿using System;
+using System.Linq;
+using System.Threading.Tasks;
+using FastTests;
+using Raven.Client.Documents.Indexes;
+using Raven.Client.Documents.Operations;
+using Raven.Client.Documents.Operations.Indexes;
+using Raven.Client.Documents.Queries;
+using Tests.Infrastructure;
+using Xunit;
+
+
+using Xunit.Abstractions;
+
+namespace SlowTests.Issues
+{
+    public class RavenDB_21233 : RavenTestBase
+    {
+        private IndexQuery q1 = new()
+        {
+            Query = @"
+from ""Orders"" update {
+    for (var i = 0; i < 10000; i++){
+        this.Freight = 13;
+    }
+}"
+        };
+
+        private IndexQuery q2 = new()
+        {
+            Query = @"
+from index ""Orders/ByCompany""
+ update {
+    for (var i = 0; i < 10000; i++){
+        this.Total = 0;
+    }
+}"
+        };
+
+        private IndexDefinition indexDefinition = new()
+        {
+            Name = "Orders/ByCompany",
+            Maps =
+            {
+                @"from order in docs.Orders
+select new
+{
+    order.Company,
+    Count = 1,
+    Total = order.Lines.Sum(l => (l.Quantity * l.PricePerUnit) * (1 - l.Discount))
+}"
+            }
+        };
+
+        public RavenDB_21233(ITestOutputHelper output) : base(output)
+        {
+        }
+
+        [Fact]
+        public async Task ExecutePatchScriptShouldRespectIgnoreMaxStepsForScriptFlag()
+        {
+            Operation operation;
+            QueryOperationOptions options = new();
+
+            using (var store = GetDocumentStore())
```

Comment:
you can change the db options here, use RavenConfiguration.GetKey when modyfing database record

Reply:
Done